xml/osc_building.xml

```diff
@@ -20,8 +20,120 @@
  
  <sect1 xml:id="sec.obs.building.spec2rpm">
   <title>From Spec Files (RPM)</title>
-  <para/>
-  <remark>TBD</remark>
+  <para>
+   To create an RPM package you need:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     the spec file
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     a file with the extension <filename class="extension">.changes</filename>
+    </para>
+   </listitem>
+   <listitem>
+    <para>the original source archive</para>
+   </listitem>
+   <listitem>
+    <para>
+     optional patches
```

Comment:
for this and the next item, maybe use "(optional)" at the end?

Reply:
I rephrased it like this:

```
optional patches which changes the original source code to fix problems
in regards to security, the build process, or other issues
```